src/lib/resource/sections/applicant-enter-your-name.js

```diff
@@ -5,41 +5,48 @@ module.exports = {
         l10n: {
             vars: {
                 lng: 'en',
-                context: {
-                    $data:
-                        '/answers/p-applicant-who-are-you-applying-for/q-applicant-who-are-you-applying-for'
-                },
                 ns: 'p-applicant-enter-your-name'
             },
             translations: [
                 {
                     language: 'en',
                     namespace: 'p-applicant-enter-your-name',
                     resources: {
-                        title: 'Enter your name',
-                        'title_someone-else': "Enter the victim's name",
+                        title: {
+                            myself: 'Enter your name',
+                            proxy: {
+                                non_deceased: "Enter the victim's name",
```

Comment:
should this be non-deceased rather than non_deceased?

Reply:
We don't have much consistency with the role naming convvention at the moment.
Example _childUnder12_ or smushed together _noauthority_ 
Could you choose one of these, that way we are not introducing a new convention.